src/Lachain.Storage/Repositories/SnapshotIndexRepository.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Linq;
+﻿using System;
```

Comment:
SnapshotIndexRepository does not seem to be the right place to handle the delete old snapshots ? What do you think about it ? Should we have a dedicated class to handle all the delete logics ?

Reply:
Created new namespace and put all the helper methods, enums and everything in there.